scripts/projects.js

```diff
@@ -51,6 +59,6 @@ projectData.forEach(function(ele) {
   projects.push(new Project(ele));
 });
 
-projects.forEach(function(ele){
-  $('#project-placeholder').append(ele.toHtml());
+projects.forEach(function(ele, index, arr){
```

Comment:
@Ansem717 try to stick with semantic and similar naming conventions - here's the one's i use:

Arrays - arr, array
Element - el, ele, or elem
Index - idx, index